test/unit/errors.test.ts

```diff
@@ -61,9 +61,9 @@ describe('errors', () => {
     // Needed because of https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work
     it('CheckCancelledByClientError should support instanceof', () => {
```

Comment:
Maybe you can also update the spelling here as its the name of the test method, you have already updated in another test test/integration-server/acrolinx-endpoint.test.ts

Reply:
I have updated the spelling here and in other places where Cancelled appeared.